hpi-swt2/bookkeeper-portal-red
# CL/feat: add personal group to user

closes #179 

To test:
User `rails c` to delete your current user. (db migrate should have added personal group automatically,too)
Log in via OIDC.
User 'rails c' to check that your current user now has a personal group.
